common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -514,7 +514,10 @@ def test_export_to_xml(self):
            <transcript language="ge" src="german_translation.srt" />
            <transcript language="ua" src="ukrainian_translation.srt" />
          </video>
-        ''')
+        """
+        # whitespace screws up etree comparison
+        expected = ''.join(line.strip() for line in expected.split('\n'))
+        expected = etree.fromstring(expected)
         self.assertXmlEqual(expected, xml)
```

Comment:
That would be fine.  This particular thing irks me a bit, because we didn't actually touch that code.  I just found a failing test and offered a fix.  Twice now.  If you require a different fix, let us know.